docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
```

Comment:
Also, it's better to add `spring.config.import=optional:file:.env[.properties]` into application.properties file 
and you will be able to use 
spring.datasource.username=${MYSQLDB_USER}
spring.datasource.password=${MYSQLDB_ROOT_PASSWORD} etc